client/src/ui/atoms/search/index.tsx

```diff
@@ -32,11 +34,29 @@ export const Search = ({
         placeholder={placeholder}
         value={value}
         disabled={isDisabled}
-        onBlur={onBlurHandler}
+        onBlur={(e) =>
+          onBlurHandler &&
+          !e.currentTarget.parentElement?.contains(e.relatedTarget) &&
+          onBlurHandler(e)
+        }
         onFocus={onFocusHandler}
         onChange={onChangeHandler}
         onClick={onClickHandler}
       />
+
+      {onResetHandler && (
+        <button
+          type="button"
+          className="button action has-icon clear-search-button"
+          onClick={onResetHandler}
+        >
+          <span className="button-wrap">
+            <span className="icon icon-cancel undefined"></span>
```

Comment:
Is that class of `undefined` intentional?

Reply:
Fixed in b66e6891c.

Reply:
(Probably not, I just copied it over.)